watson_developer_cloud/nlu/features/v1/__init__.py

```diff
@@ -0,0 +1,72 @@
+class Feature(object):
```

Comment:
you have to use the staging url for it to work.  Yes, you are correct, it type checks the features  param (which is a list of feature subclasses).